packages/components/src/components/card/card.scss

```diff
@@ -18,6 +18,18 @@
   width: 350px;
   height: auto;
 
+  // &.borderHovered { 
+  //   border-color: #ebe9e9;
+  // }
+
+  &.linkHovered { 
+    border-color: #ebe9e9;
```

Comment:
this color actually doesnt exist in our tokens. i think its a wrong grey tone and should be #EEEDED (engineering 200)